pkg/api/v20160330/validate.go

```diff
@@ -71,6 +71,15 @@ func (l *LinuxProfile) Validate() error {
 
 // Validate implements APIObject
 func (a *Properties) Validate() error {
+	if a.OrchestratorProfile == nil {
+		return fmt.Errorf("missing OrchestratorProfile")
+	}
```

Comment:
I'm these versions. If this isn't specified we default to a DCOS cluster. No error should be returned as it is a valid api model to not have orchestrator profile.